.travis.yml

```diff
@@ -36,6 +36,8 @@ matrix:
   include:
     - rvm: 2.3.3
       gemfile: gemfiles/Gemfile.shopify
+    - rvm: 2.4.0
```

Comment:
why not put 2.4.4?

Reply:
simpy because we don't have it included in rvm, just didn't want to extend the build time that much... but I could

Reply:
well, don't even know if it would, gotta try it out